organization/annual-reports/financial-report-2021.md

```diff
@@ -0,0 +1,559 @@
+---
+type: resource
+---
+
+
+
```

Comment:
lines 4, 5, and 6 are all blank. I expect those should be collapased to a single blank line

Reply:
My delinter didn't pick that up!